contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--1.2.0--1.3.0.sql

```diff
@@ -914,5 +914,14 @@ AS $$
 $$
 LANGUAGE SQL IMMUTABLE PARALLEL RESTRICTED;
  
+CREATE TABLE sys.babelfish_view_def (
+	db_name NAME NOT NULL,
+	schema_name NAME NOT NULL,
+	object_name sys.NVARCHAR(128) NOT NULL,
+	definition TEXT NOT NULL COLLATE "C",
+	PRIMARY KEY(db_name, schema_name, object_name)
+);
+GRANT SELECT ON sys.babelfish_view_def TO PUBLIC;
+
```

Comment:
What is our policy for views created before this change? Are we going to keep those entries with null view definition or just to ignore them?

Reply:
For now, there wont be any entires for views prior to this change. We can ask the User to either drop and re-create the view they are blocked on or we can provide some other way to insert their view definition into this table (The first workaround seems to be the better one)